src/components/molecules/EventModal/EventModal.tsx

```diff
@@ -45,12 +46,16 @@ const EventModal = ({
       if (!isEqual(currentEvent, inputFieldText) && showEventModal) {
         setIsAlertOpen(true);
       }
+    } else if (unsavedEvents.hasOwnProperty(selectedDate)) {
```

Comment:
**Suggestion:** Did you know you can do:
```
} else if ('selectedDate' in unsavedEvents) {
```

Just found I'd mention it in case you prefered it as I prefer how it looks 🤷‍♂️  😅